portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -57,9 +57,32 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
   public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
     String comment = request.getParameter("comment");
     // TODO(meagancheese): Add Sanitization Step
+    int max = getMax(response);
+    if (max == -1) {
+      response.setContentType("text/html");
+      response.getWriter().println("Please enter a valid integer number greater than zero.");
+      return;
+    }
     Entity commentEntity = new Entity("Comment");
     commentEntity.setProperty("text", comment);
     datastore.put(commentEntity);
     response.sendRedirect("/index.html");
   }
-}
+  
+  private int getMax(HttpServletRequest request) {
+    String maxString = request.getParameter("max");
+    int max;
+    try{
+      max = Integer.parseInt(maxString);
```

Comment:
If you drop the validation after the catch, you can simplify this logic by just returning Integer.parseInt(maxString) directly, and eliminating the `int max;` above.